pkg/storage/bloom/v1/tokenizer.go

```diff
@@ -1,15 +1,40 @@
 package v1
 
 import (
+	"fmt"
 	"unicode/utf8"
 
+	"github.com/grafana/loki/pkg/push"
 	iter "github.com/grafana/loki/v3/pkg/iter/v2"
 )
 
 const (
 	MaxRuneLen = 4
 )
 
+type StructuredMetadataTokenizer struct {
+	prefix string
```

Comment:
nit: I'd add what the prefix typically is here:
```suggestion
        // prefix to add to the tokens: typically the chunkref
	prefix string
```